frinx-uniconfig/user-guide/network-management-protocols/uniconfig-installing/readme.md

```diff
@@ -658,6 +658,285 @@ curl --location --request POST 'http://localhost:8181/rests/operations/connectio
+request inside of `gnmi-topology:credentials` container.
+
+- **gnmi-topology:username** - Name of the user that has permission
+    to access device using gNMI.
+- **gnmi-topology:password** - Password to the user.
+
+### Session timers
+
+The following parameters adjust timers that are related with maintaining
+of gNMI session state. None of these parameters are mandatory
+(default values will be used).
+
+- **gnmi-topology:request-timeout** - Timeout for each gnmi request. Request times out if not completed in X seconds (default value: 30 s).
+
+### Flags
+
+Non-mandatory flag parameters that can be added to mount-request.
+
+- **gnmi-topology:enabled-notifications-** - If it is set to 'true' and gNMI device supports notifications, gNMI mountpoint will
+    expose GNMI notification and subscription services. (default value: true).
+
+### Other parameters
+
+Other non-mandatory parameters that can be added to mount-request.
+
+- **gnmi-topology:dry-run-journal-size** - Size of the DRY RUN gNMI mountpoint journal. DRY RUN journal captures gNMI operations 
+    that would be executed when reading/writing some configuration. However the operations are not actually
+    sent to the device (default value: 0).
+
+### Extension parameters
+
+Other extended non-mandatory parameters that can be added to mount-request inside of `extensions-parameters` container.
+
+#### gNMI parameters
+
+- **gnmi-topology:use-model-name-prefix** - Some devices require a module prefix in first element name
+    of gNMI request path (e.g interfaces -> openconfig-interfaces:interfaces) (default value: false).
+
+### UniConfig-native
+
+Parameters related to installation of gNMI node with
+uniconfig-native support.
+
+- **uniconfig-config:uniconfig-native-enabled** - Whether
+    uniconfig-native should be used for installation of NETCONF, CLI or gNMI
+    node. By default, this flag is set to 'false'.
+- **uniconfig-config:sequence-read-active** - Forces reading of data
+    sequentially when mounting device. If it is set to 'true' sync-from-network 
+    will be done in parallel. By default, this flag is set to 'false'.
+- **uniconfig-config:whitelist** - List of root YANG entities that should be read.
+- **uniconfig-config:store-failed-installation** - Whether the installation
+    should be stored in the database if it fails (e.g. is unreachable).
+    The node will be 'installed' even though it failed and the user has 2 options:
+  - uninstall the device and reinstall it.
+  - call sync-from-network to sync the data from the device.
+
+!!!
+Important install parameter is 
+**gnmi-topology:schema-cache-directory**: **<folder-name>**. It specified folder name 
+in cache directory with YANG schemas needed for installation of device. 
+!!!
+
+### Update paths
+
+This is non-mandatory parameter that specified list of paths to subtree that 
+UniConfig will process the intended change of it as gNMI SET message - Update operation. 
```

Comment:
"This is non-mandatory parameter that specified list of paths to subtree that UniConfig will process the intended change of it as gNMI SET message - Update operation."

Is there another way of phrasing this?

Reply:
rephrased